vyper/codegen/stmt.py

```diff
@@ -414,9 +414,6 @@ def _is_terminated(code):
 
 # codegen a list of statements
 def parse_body(code, context, ensure_terminated=False):
-    if not isinstance(code, list):
```

Comment:
what validation shows this is not used anymore?

Reply:
it's not in the coverage report. you can also check all the calls to parse_body and verify that they pass lists